### PR TITLE
MailChannels Pages Plugin

### PR DESCRIPTION
**Pull request by GregBrimble**
cc @evanderkoogh

### CODE REVIEW

File: content/pages/platform/functions/plugins/mailchannels.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+- `subject` is an optional `String` or function which returns a string. It defaults to `New <name> form submission`.
+
+- `content` is an optional array or function which returns an array. It defaults to a `text/html` and `text/plain` body array, detailing the form submission contents.
+
+- `respondWith` is a required function which takes the `request`, `formData` and `name` object parameter and returns a `Response` or `Promise` of a `Response`. Assuming the form submission is successful, this function will be called to generate response for visitors.
+
+The `method` and `action` attributes of the HTML form do not need to be set. The Plugin will automatically override them to allow it to intercept the submission.
+
+For more information about MailChannels and the options they support, check out [their documentation](https://mailchannels.zendesk.com/hc/en-us/articles/4565898358413-Sending-Email-from-Cloudflare-Workers-using-MailChannels-Send-API).
```
Comment (deadlypants1973) [inline content/pages/platform/functions/plugins/mailchannels.md]:
```suggestion
For more information about MailChannels and the options they support, refer to [the documentation](https://mailchannels.zendesk.com/hc/en-us/articles/4565898358413-Sending-Email-from-Cloudflare-Workers-using-MailChannels-Send-API).
```

File: content/pages/platform/functions/plugins/mailchannels.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+</body>
+```
+
+The Plugin takes a single argument, an object with a number of properties:
+
+- `personalizations` is required and should be set to either an array, or a function which returns an array. If set to a function, it is passed a single object parameter which contains the incoming `request` (`Request`), `formData` (`FormData`) and `name` (`String`). This is useful if you want to populate dynamic values based on the form submission.
+
+- `from` is also mandatory and should be set to either an object, or a function which returns an object. Again, if set to a function, it is passed a single object parameter which contains `request`, `formData` and `name`.
+
+- `subject` is an optional `String` or function which returns a string. It defaults to `New <name> form submission`.
```
Comment (deadlypants1973) [inline content/pages/platform/functions/plugins/mailchannels.md]: If <name> is generated by an input value the user decides, then per CF style guide, we should change to "<NAME>".

**Review by deadlypants1973 — APPROVED** [review]
PCX approved with comment

File: content/pages/platform/functions/plugins/mailchannels.md (hunk 2 above)
Comment (GregBrimble) [inline content/pages/platform/functions/plugins/mailchannels.md]:
```suggestion
- `subject` is an optional `String` or function which returns a string. It defaults to `New <NAME> form submission`.
```

**Pull request merged**